lib/ansible/utils/module_docs_fragments/mysql.py

```diff
@@ -66,9 +66,9 @@ class ModuleDocFragment(object):
    - PyMySQL (Python 2.7 and Python 3.X), or
    - MySQLdb (Python 2.x)
 notes:
-   - Requires the PyMySQL (Python 2.7 and Python 3.X) or MySQL-python (Python 2.X) Python package on the remote host.
-     For Ubuntu, this is as easy as apt-get install python-pymysql. (See M(apt).) For CentOS/Fedora, this
-     is as easy as yum install python2-PyMySQL. (See M(yum).)
+   - Requires the PyMySQL (Python 2.7 and Python 3.X) or MySQL-python (Python 2.X) package on the remote host.
+     The Python package may be installed with apt-get install python-pymysql (Ubuntu; see M(apt)) or
+     yum install python2-PyMySQL (RHEL/CentOS/Fedora; see M(yum)).
```

Comment:
Newer Fedora versions uses `dnf` by-default. It may be worth adding that as well if you are editing in these lines. What say ?

Reply:
Rest LGTM

Reply:
Sorry, I don't feel familiar enough with Fedora/dnf (just read that it's "the future yum"), it would be safer for somebody else to clarify that.